e2e/Cargo.toml

```diff
@@ -12,6 +12,7 @@ hex-literal = "0.3"
 lazy_static = "1.4"
 model = { path = "../model" }
 orderbook = { path = "../orderbook" }
+prometheus = "0.12"
```

Comment:
Does the fact that we are adding the metrics in the e2e test mean we only measure it when tests run?

Reply:
No. Just need to import this crate to create an instance of Registry.